Configs/.local/lib/hyde/weather.py

```
@@ -325,6 +325,15 @@ def get_default_locale() -> tuple[str, TempUnit, TimeFormat, WindUnit]:
 windspeed_unit: WindUnit = "km/h"
 
 
+def get_weather_data(url: str, headers: dict[str, str]) -> WttrResponse | None:
+    try: return cast(WttrResponse, requests.get(url, timeout=10, headers=headers).json())
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)